public/js/reducers/breakpoints.js

```diff
@@ -123,6 +123,9 @@ function update(state = State(), action: Action) {
       }
 
       break;
+    }
+    case "NAVIGATE": {
+      return State();
```

Comment:
We don't actually want to do this, for Firefox at least. We keep breakpoints around, just like the server does. If you navigate back to a source, all the breakpoints should still exist. A key use case is refreshing: all the breakpoints should still exist, right? A refresh is a type of navigation.

Not sure if you're seeing some problem in Chrome, but I'm pretty sure this will break Firefox. (it'll pause on a top-level breakpoint after a refresh but the breakpoint won't exist in the UI)